glossary.yml

```diff
@@ -139,13 +139,6 @@
     def: >
       The difference between the measured or inferred value of a quantity and its actual value, 
       sometimes with the absolute value taken. 
-  it:
```

Comment:
this PR seems to only remove an entry instead of adding a new one. Was this a git accident?